### DISCUSSION TITLE
Is there a way to simultaneously bind and set a `TypeVar` default with PEP 695 syntax?

### DISCUSSION THREAD

**Discussion by max-muoto** [microsoft/pyright#8210] [Q&A] [answered]
Is there a way that you can set both a bound, and a default with PEP 695 syntax with 3.13 type var defaults? I haven't seen any examples of this, so was curious if it was even possible.

**Comment by erictraut** [comment, marked as answer]
Yes, you can set both a bound and a default, like this:

```python
class Foo[T: float = int]: ...
```
